Chocolatey GUI 0.15.0 Blog Post

gep13 opened the pull request:
Blog Post for the new release of Chocolatey GUI.

gep13 commented:
@ferventcoder @RichiCoder1 @pascalberger @mwallner @AdmiringWorm what are your collective thoughts on this?

Not going to have as much time as I thought today to do this as we are heading out for the weekend, but I think I covered the main points.

Anything that I forgot?  Wrote this whilst trying to give breakfast to my two girls, so I expect there to be some typos/grammar errors, so feel free to point them out 😄

RichiCoder1 commented on chocolatey/Website/Views/Blog/ChocolateyGUI0.15.0.md:
```diff
@@ -0,0 +1,59 @@
+
+## New Branding
+
+When we switched to using WPF as the base framework for Chocolatey GUI, we introduced what we thought was a sensible colour scheme, emphasing on the "chocolatey" theme.  Looking back, we think we went a little too far.  In the 0.15.0 release, we have changed the branding again, which is more in-keeping with the overall Chocolatey branding.
```
Hatin' on my chocolate bar theme ;P

RichiCoder1 commented on chocolatey/Website/Views/Blog/ChocolateyGUI0.15.0.md:
```diff
@@ -0,0 +1,59 @@
+
+## Chocolatey Lib
+
+In previous versions of Chocolatey GUI, we shelled out directly to PowerShell to invoke the various commands that are available within Chocolatey.  In this release, we are now using the official [chocolatey.lib](https://www.nuget.org/packages/chocolatey.lib), which means we now have a much better integration with the features that Chocolatey offers.  In doing this refactoring, we have also elimin ...
```
It'd be also worth calling out that perf was great improved with this release.

gep13 replied:
Done

pascalberger commented on chocolatey/Website/Views/Blog/ChocolateyGUI0.15.0.md:
```diff
@@ -0,0 +1,59 @@
+
+When we switched to using WPF as the base framework for Chocolatey GUI, we introduced what we thought was a sensible colour scheme, emphasing on the "chocolatey" theme.  Looking back, we think we went a little too far.  In the 0.15.0 release, we have changed the branding again, which is more in-keeping with the overall Chocolatey branding.
+
+If you are interested in seeing the evolution of the branding for this release, you can follow along in this [commit](https://github.com/chocolatey/ChocolateyGUI/commit/ed894cc4e16abe5d33de1275efae49803d8d1919) and then this [commit](https://github.com/chocolatey/ChocolateyGUI/commit/c7a92e35a4b7e43cc611d7f8fd854ba8bb3171e2#diff-bcb0247f0d40fae191239e76914070a0).
```
Instead of linking to commits I would embed images for a certain screen. Since this is already the "read more" version I prefer to not click further to get additional information.

gep13 replied:
Done

pascalberger commented on chocolatey/Website/Views/Blog/ChocolateyGUI0.15.0.md:
```diff
@@ -0,0 +1,59 @@
+
+If you are interested in seeing the evolution of the branding for this release, you can follow along in this [commit](https://github.com/chocolatey/ChocolateyGUI/commit/ed894cc4e16abe5d33de1275efae49803d8d1919) and then this [commit](https://github.com/chocolatey/ChocolateyGUI/commit/c7a92e35a4b7e43cc611d7f8fd854ba8bb3171e2#diff-bcb0247f0d40fae191239e76914070a0).
+
+We would love to hear any feedback that you might have related to the new branding.
```
Provide way how to send feedback

gep13 replied:
Done

pascalberger commented on chocolatey/Website/Views/Blog/ChocolateyGUI0.15.0.md:
```diff
@@ -0,0 +1,59 @@
+
+We know that Chocolatey GUI is used across the world, by speakers of lots of different languages.  With this in mind, the decision was taken to support translation of all the major parts of Chocolatey GUI.  This will be an ongoing effort, however, in this release, there is now support for English, Norwegian, German and Swedish.
+
+If you are interested in helping support your language in Chocolatey GUI, then please reach out via an issue on the GitHub repository.
```
Make GitHub repository a link to the repository. Or maybe even directly to the new issue form.

gep13 replied:
Done

mwallner commented:
great post, please try to integrate pascal's notes :-)

gep13 commented:
@RichiCoder1 @ferventcoder @pascalberger @mwallner I believe that I have addressed all the comments.

Please could you review the changes and let me know if there is anything else?

Thanks

gep13 commented:
@ferventcoder doh! I keep forgetting about the split between the documents. 😢
